libraries/standard/json/src/json.c

```diff
@@ -366,7 +365,9 @@ static bool_ skipEscape( const char * buf,
     bool_ ret = false;
     size_t i = *start;
 
```

Comment:
`assert( *start < max )`?

Reply:
Asserts for NULL check of `buf`?

Reply:
Would it make sense to add these asserts everywhere else then? They seem a little redundant since we are making those asserts in the public-facing function. When does it make sense to have an assert @dan4thewin ?